mozregression/cli.py

```diff
@@ -390,6 +418,7 @@ def validate(self):
         options.bits = parse_bits(options.bits or mozinfo.bits)
         fetch_config = create_config(options.app, mozinfo.os, options.bits)
         self.fetch_config = fetch_config
+        self.fetch_config.artifact_name = options.artifact_name
```

Comment:
hm, could you move that next to fetch_config.set_build_type(options.taskcluster) in the special case taskcluster handling ?
also a fetch_config.set_artifact_name would make sense, I mean we will set the two options in the same way, and at the same place.